recipes/snap/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+make
+mkdir -p $PREFIX/bin
+mkdir -p ${PREFIX}/share/snap
+mkdir -p ${PREFIX}/etc/conda/activate.d ${PREFIX}/etc/conda/deactivate.d
+cp -p snap fathom forge hmm-info exonpairs cds-trainer.pl hmm-assembler.pl noncoding-trainer.pl patch-hmm.pl zff2gff3.pl $PREFIX/bin
+cp -pr HMM ${PREFIX}/share/snap
+cp -pr DNA ${PREFIX}/share/snap
+
+# set environment variables using conda's environment activate/deactivate
```

Comment:
Yes, having wrappers that just forward the args via `$@` is the way to go I think.